narayana-bom-test/pom.xml

```diff
@@ -17,7 +17,7 @@
   <name>Narayana Bom Test</name>
   <description>Narayana BOM is for centralising dependency management for development purposes (test scoped dependencies)</description>
 
-<!--
+  <!--
 	This Bill of Materials (BOM) is for centralising dependency management across the various modules of Narayana.
     The BOM is intended for development purposes within Narayana.
 -->
```

Comment:
shouldn't we add spaces here as well?

Reply:
I don't know how this one ended up in the commit, I'll remove it.

Reply:
Found it, by default we [enable sortpom](https://github.com/jbosstm/narayana/blob/7.0.0.Final/pom.xml#L178). Is that a bug or a feature (I'm sure we've discussed it before but any new pom should verify that it passes sort pom so we all need to be vigilant to check that during reviews).